src/stories/BaseComponents/robot-component.tsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import { actions } from '@storybook/addon-actions';
+
+import Robot, { RobotProps } from '../../components/schedule-visualizer/robot';
+import { viewBoxCoords, componentDisplayStyle } from './Utils';
+
+interface RobotComponentProps extends RobotProps {
+  renderInfoPanel(): JSX.Element;
+}
+
+export default function RobotComponent(props: RobotComponentProps): React.ReactElement {
+  const { robot, footprint, colorManager, renderInfoPanel } = props;
+
+  return (
+    <div style={componentDisplayStyle.display}>
+      {renderInfoPanel()}
+
+      <svg viewBox={viewBoxCoords}>
+        <Robot
+          robot={robot}
+          footprint={footprint}
+          colorManager={colorManager}
+          {...actions('onClick')}
```

Comment:
Clicking the robot causes 2 events to fire, can you investigate this?

Reply:
There are 2 `onClicks` within `robots.tsx`, one on the `g` element and another on the `circle` element. I removed the one on `circle` as it fails the unit tests and is inconsistent with the other components.